public/assets/Fork-Awesome-1.1.7/.gitignore

```diff
@@ -0,0 +1,15 @@
+.DS_Store
```

Comment:
and if pulling them in is not possible and they are git projects, please use git submodules.